src/controllers/connectionManager.ts

```diff
@@ -783,6 +783,8 @@ export default class ConnectionManager {
         } else if (selection === LocalizedConstants.readMore) {
             this.vscodeWrapper.openExternal(Constants.encryptionBlogLink);
             await this.showInstructionTextAsWarning(profile, reconnectAction);
+        } else if (selection === LocalizedConstants.Common.cancel) {
+            await reconnectAction(undefined);
```

Comment:
Probably should have a comment (or somewhere in the contract) on IReconnectAction to state that passing `undefined` should mean a cancellation.

```suggestion
            await reconnectAction(undefined /* profile */);
```